pkg/skaffold/debug/util/util.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
can this be called `annotations` package instead of `util` (and corresponding file and folder names)?
The import would look neat as
```go
import "github.com/GoogleContainerTools/skaffold/pkg/skaffold/debug/annotations"
```